pilot/pkg/networking/core/v1alpha3/envoyfilter.go

```diff
@@ -203,7 +203,7 @@ func listenerMatch(in *plugin.InputParams, listenerIP net.IP,
 }
 
 func insertHTTPFilter(listenerName string, filterChain *listener.FilterChain, hcm *http_conn.HttpConnectionManager,
-	envoyFilter *networking.EnvoyFilter_Filter) {
+	envoyFilter *networking.EnvoyFilter_Filter, is11 bool) {
```

Comment:
nit: `is11` is not really descriptive, `use_typed_config` or something? (throughout whole PR)

Reply:
it's only temporary. it's just propagating node metadata, since we really have no other way to detect proxy capabilities.